GooglePlayInstant/Samples/SphereBlast/scripts/PlayInstant.cs

```diff
@@ -0,0 +1,178 @@
+﻿// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using GooglePlayInstant;
+
+public class PlayInstant : MonoBehaviour
```

Comment:
I think that you can delete this file once the Cookie API PR is merged.

Reply:
That should be the case. I'll hold off for now until the Cookie API PR is merged.